Rakefile

```diff
@@ -23,7 +23,7 @@ end
 require "yardstick/rake/verify"
 Yardstick::Rake::Verify.new do |verify|
   verify.require_exact_threshold = false
-  verify.threshold = 58
+  verify.threshold = 58.1
```

Comment:
It is. And `verify.threshold = 58.1` in conjunction with `verify.require_exact_threshold = false` (above) means `>= 58.1`. We bump it up to try to enforce ourselves to "keep moving forward" to the 100% API documentation coverage :D